### PR TITLE
drone-kustomize: add optional param rollout_timeout

### PR DESCRIPTION
**Pull request by razielgn**

### CODE REVIEW

File: drone-kustomize/plugin.sh [hunk 1]
```diff
@@ -27,5 +27,5 @@ fi
 kustomize build | kubectl apply -f -
 
 if [ -n "$PLUGIN_ROLLOUT_DEPLOYMENT" ];then
-  kubectl rollout status deployment $PLUGIN_ROLLOUT_DEPLOYMENT -n $PLUGIN_ROLLOUT_NAMESPACE --timeout=180s
-fi
\ No newline at end of file
+  kubectl rollout status deployment $PLUGIN_ROLLOUT_DEPLOYMENT -n $PLUGIN_ROLLOUT_NAMESPACE --timeout=${PLUGIN_ROLLOUT_TIMEOUT:-"180s"}
```
Comment (angelbarrera92) [inline drone-kustomize/plugin.sh:30]: Default value should be `0s` to don't disrupt the current behavior. After that, LGTM!

Comment (razielgn) [reply drone-kustomize/plugin.sh:30]: Uh, previously `180s` was hardcoded. Are you asking me to put `0s` as the default fallback?

Comment (angelbarrera92) [reply drone-kustomize/plugin.sh:30]: OK! Sorry! You're right. LGTM!

**Comment by razielgn** [comment]
Thank you!

**Pull request merged**